internal/nfs/controller/volume.go

```diff
@@ -134,25 +135,57 @@ func (nv *NFSVolume) CreateExport(backend *csi.Volume) error {
 		return fmt.Errorf("failed to set NFS-cluster: %w", err)
 	}
 
-	// ceph nfs export create cephfs ${FS} ${NFS} /${EXPORT} ${SUBVOL_PATH}
-	args := []string{
-		"--id", nv.cr.ID,
-		"--keyfile=" + nv.cr.KeyFile,
-		"-m", nv.mons,
-		"nfs",
-		"export",
-		"create",
-		"cephfs",
-		fs,
-		nfsCluster,
-		nv.GetExportPath(),
-		path,
+	// TODO: use new go-ceph API, see ceph/ceph-csi#2977
```

Comment:
do we know when go-ceph release is planned, is it this week ?

Reply:
> do we know when go-ceph release is planned, is it this week ?

It should be around april second/third week. 

We probably need a Ceph pacific release before pr for nfs export management goes into go-ceph. 
We'll not be getting go-ceph bindings anytime soon.